pages/index.js

```diff
@@ -1,69 +1,218 @@
-import Head from 'next/head'
```

Comment:
We should use the Head on the mainpage to put in the title etc. (this is also a test comment for my first review)